README.md

```diff
@@ -23,41 +23,54 @@ Officially, `TNoodle-WCA` is a [scramble program](https://www.worldcubeassociati
 
 ## Project Details
 
-`tmt` (TNoodleMakeTools) is a python script used to develop TNoodle.
+TNoodle is organised as a multi-project [Gradle](https://gradle.com) build. The build files are written in the type-safe `Kotlin` dialect.
+
+Every sub-project has its individual artifact configuration and `build.gradle` file. Furthermore, there is a central `buildSrc` folder,
+which is automatically sourced by Gradle. It contains common code and shared configuration setups.
 
 
 ### Overview
 
-Get a high level view of all the projects that comprise tnoodle by running:
+Gradle is served through the use of a `Gradle wrapper` available as `gradlew` (UNIX systems) or `gradlew.bat` (that other operating system nobody wants to use)
+It is recommended to set up an alias to simplify task generation, along the lines of `alias gw='./gradlew --parallel'`.
+
+Get an overview of the core project tasks by executing
+
+    ./gradlew tasks
 
-    ./tmt graph --descriptions
+If you want fancy console output, set the option `org.gradle.console` to value `rich`.
+This setting is currently not checked in because it makes Travis CI logs horrible to read, but you can easily include it by modifying (or creating, if it doesn't exist yet) `~/.gradle/gradle.properties` to something like
+
+    org.gradle.console=rich
 
 
 ### WCA Scramble Program
 
 When you're ready to develop, run the following and then visit <http://localhost:2014/scramble/>
 
-    ./tmt make run -p wca
+    ./gradlew :webscrambles:runShadow
```

Comment:
Should this command work without setting up android? When I try it, I get the following output:

```
➜  ~/gitting/tnoodle git:(9254e38c) ./gradlew :webscrambles:runShadow

FAILURE: Build failed with an exception.

* Where:
Build file '/home/jeremy/gitting/tnoodle/build.gradle.kts' line: 31

* What went wrong:
A problem occurred configuring project ':tnoodle-android'.
> SDK location not found. Define location with an ANDROID_SDK_ROOT environment variable or by setting the sdk.dir path in your project's local properties file at '/home/jeremy/gitting/tnoodle/local.properties'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
```

I'd really like for it to be possible for developers to do non-android things and not have to install the android SDK.

Reply:
That sounds nice indeed, but I'm afraid Gradle needs android to do some basic build configurations. There is no easy workaround that doesn't involve hijacking Gradle (which you don't want to, believe me).

All I can offer you as a "fix" is moving `tnoodlejs` and `tnoodle-android` to a separate repository (i.e. project) as soon as possible after merging this PR. That would also mean removing the Android plugin from the core Gradle build.

Reply:
SGTM!